src/features/usage-consolidation-accordion/titles-table/titles-table.js

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  useIntl,
+  FormattedNumber,
+} from 'react-intl';
+import { Link } from 'react-router-dom';
+
+import {
+  MultiColumnList,
+  NoValue,
+} from '@folio/stripes/components';
+
+import LoadingMessage from '../loading-message';
+import Toaster from '../../../components/toaster';
+import {
+  formatCost,
+  formatValue,
+} from '../utilities';
+import { useMultiColumnListSort } from '../../../hooks';
+import {
+  costPerUse as costPerUseShape,
+  costPerUseTypes,
+  sortOrders,
+} from '../../../constants';
+
+const propTypes = {
+  costPerUseData: costPerUseShape.CostPerUseReduxStateShape.isRequired,
+  fetchNextPage: PropTypes.func.isRequired,
+  onSortTitles: PropTypes.func.isRequired,
+};
+
+const PAGE_SIZE = 100;
+
+const TitlesTable = ({
+  costPerUseData,
+  fetchNextPage,
+  onSortTitles,
+}) => {
+  const intl = useIntl();
+  const [page, setPage] = useState(1);
```

Comment:
No magic number, probably `1` should be changed by, for exp:  `const DEFEAULT_PAGE = 1`;